tests/functional/test_functional.py

```diff
@@ -156,3 +156,7 @@ def test_bot_reply_with_unicode_message(driver):
     driver.wait_for_bot_channel_message(u'你好!')
     driver.send_channel_message(u'你不明白，对吗？')
     driver.wait_for_bot_channel_message(u'.*You can ask me.*')
+
+def test_bot_reply_with_alias_message(driver):
+    driver.send_channel_message("! hello", False, False)
+    driver.wait_for_bot_channel_message("hello sender!", tosender=True)
```

Comment:
this is useful to have here though @migetman9 we need to ensure something sets ALIASES either here in the code for the driver, or in whatever script runs the functional testing